src/appleseed/renderer/modeling/object/curveobjectreader.h

```diff
@@ -57,25 +57,6 @@ class APPLESEED_DLLSYMBOL CurveObjectReader
         const foundation::SearchPaths&  search_paths,
         const char*                     name,
         const ParamArray&               params);
-
-  private:
-    static foundation::auto_release_ptr<CurveObject> create_hair_ball(
```

Comment:
Where has this code been moved to?

Reply:
Oh I was under the impression that it didn't need to be supported by the new curve object, but i've added it back in for builtin testing.